lib/mapper.js

```diff
@@ -24,7 +24,17 @@ exports.identify = function(identify){
   var userId = identify.userId();
   var anonymousId = identify.sessionId();
   var payload = {};
+  var ip = identify.ip();
+  var shouldTrackIp = identify.proxy('context.Kissmetrics.trackIp');
```

Comment:
This shouldn't be inside the context. It should be an option in the UI instead of requiring users to implement custom code for kissmetrics.